data/piper.1.in

```diff
@@ -0,0 +1,22 @@
+.TH PIPER 1 "@version_date@" "@version@" "User Commands"
+.SH NAME
+Piper \- GTK application to configure gaming devices
+.SH DESCRIPTION
+.SS "Usage:"
+.IP
+piper [OPTION]
+.SS "Help Options:"
+.TP
+\fB\-h\fR, \fB\-\-help\fR
+Show help options
+.TP
+\fB\-\-help\-all\fR
+Show all help options
+.TP
+\fB\-\-help\-gapplication\fR
+Show GApplication options
+.TP
+\fB\-\-help\-gtk\fR
+Show GTK+ Options
```

Comment:
as said above, none of these options are under our control and they're of questionable use to most users anyway. Let's not document them (except `--help` obviously).